ledger-tool/src/main.rs

```diff
@@ -1146,6 +1218,30 @@ fn main() {
                             "geyser_plugin_config",
                         ])
                         .help("In addition to the bank hash, optionally include accounts and/or transactions details for the slot"),
+                )
+                .arg(
+                    Arg::with_name("abort_on_invalid_block")
+                        .long("abort-on-invalid-block")
+                        .takes_value(false)
+                        .help(
+                            "Exits with failed status early as soon as any bad block is detected",
+                        ),
+                )
+                .arg(
+                    Arg::with_name("no_block_cost_limits")
+                        .long("no-block-cost-limits")
+                        .takes_value(false)
+                        .help("Sets block cost limits to the max"),
+                )
+                .arg(
+                    Arg::with_name("enable_hash_overrides")
+                        .long("enable-hash-overrides")
```

Comment:
when would we ever not want this?

Reply:
I think I'm still not understanding.

- In simulation, we always want this.
- In other modes, we never want this.

are above 2 points incorrect?

Reply:
>     * In simulation, we always want this.

yes. we want this for `ledger-tool simulate-block-production`.

>     * In other modes, we never want this.

yes and no. when verifying simulated blocks, we want this. when verifying normal blocks, we never want this.

Thus, `--enable-hash-overrides` is an opt-in not to change the normal operations of `ledger-tool verify`.